nat-lab/tests/utils/analytics/event_loader.py

```diff
@@ -80,6 +76,11 @@ def fetch_moose_events(database_name):
         for event in events:
             event_json = json.loads(event[0])
             flatten_event_json = flatten(event_json)
+
+            # Skip init event
+            if flatten_event_json["event_name"] == "init":
+                continue
+
```

Comment:
Is this needed? I assume the new moose tracker emits this event from now on, however the python code is used for the mocked moose code which might not emit this event(i might be wrong just trying to be sure)

Reply:
No, nat-lab tests use the real moose tracker.
And yes as far as I saw this event always appear with the new tracker.

Reply:
NatLab using real tracker is very good. Might I offer an improvement for actually testing that this event is present and if not - then raising a fault?

Reply:
Hmm, could be a good idea